app/editor/package.json

```diff
@@ -23,7 +24,7 @@
     "react-dom": "17.0.2",
     "react-error-boundary": "3.1.4",
     "react-icons": "4.3.1",
-    "react-redux": "7.2.6",
+    "react-redux": "7.2.8",
     "react-redux-loading-bar": "5.0.2",
     "react-router-dom": "6.2.2",
```

Comment:
Note the latest version of `react-router-dom` is broken.  I think it was `6.3.0` or something like that.